src/mgr/ActivePyModules.cc

```diff
@@ -63,67 +63,6 @@ ActivePyModules::ActivePyModules(
 
 ActivePyModules::~ActivePyModules() = default;
 
-namespace {
```

Comment:
This seems somewhat unrelated to this PR. you want to split it up into a new PR?

Reply:
i created #39610. but that PR conflicts with this one. i would like to get it merged first.